### PR TITLE
Readthedocs fix

### LINKED ISSUE #97: Readthedocs
Need to get better public facing documentation of our code.

### PR DESCRIPTION
**Pull request by aelanman**
It's building on rtd now. I made this branch's version visible on pyuvsim.readthedocs.io if you want to check it out.

Also make `line_profiler` an optional dependency.

fixes #97 
addresses part of #106 (also needs updates to setup.py)

### CODE REVIEW

File: ci/appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Doesn't work because _miriad doesn't compile on Windows.
```
Comment (bhazelton) [inline ci/appveyor.yml]: appveyor is the service to do continuous integration testing on a windows platform. I'd take this file out unless you really want to go this route. Note that pyuvdata currently doesn't work on windows.

File: ci/install.ps1 [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Sample script to install Miniconda under Windows
```
Comment (bhazelton) [inline ci/install.ps1]: this file is also appveyor-related. I'd remove it.

File: ci/run_with_env.cmd [hunk 3]
```diff
@@ -0,0 +1,94 @@
+:: EXPECTED ENV VARS: PYTHON_ARCH (either x86 or x64)
```
Comment (bhazelton) [inline ci/run_with_env.cmd]: this file is also appveyor-related. I'd remove it.

**Comment by bhazelton** [comment]
@aelanman for coverage issue, I don't think we care if the the case where line_profiler is not installed is covered by tests, so I'd add this comment on the else line: `# pragma: no cover`

**Comment by aelanman** [comment]
I was just about to ask! Thanks.

**Pull request merged**